src/index.js

```diff
@@ -24,18 +24,28 @@ taskDescriptionInput.addEventListener('keypress', (e) => {
 });
 
 clearAll.addEventListener('click', () => {
-  const uncompleted = [];
-  allTasks.forEach((task, index) => {
-    if (!task.completed) {
-      uncompleted.push(task);
-      removeTaskFromDisplay(index + 1);
-    }
-  });
-
+  const uncompleted = allTasks.filter((task) => !task.completed);
   uncompleted.forEach((task, index) => {
     task.index = index + 1;
   });
   taskObj.tasksArr = uncompleted;
   taskObj.updateLocalStorage();
   window.location.reload();
 });
+
+// clearAll.addEventListener('click', () => {
+//   const uncompleted = [];
+//   allTasks.forEach((task, index) => {
+//     if (!task.completed) {
+//       uncompleted.push(task);
+//       removeTaskFromDisplay(index + 1);
+//     }
+//   });
+
+//   uncompleted.forEach((task, index) => {
+//     task.index = index + 1;
+//   });
+//   taskObj.tasksArr = uncompleted;
+//   taskObj.updateLocalStorage();
+//   window.location.reload();
+// });
```

Comment:
Kindly remove all the commented code to make your code base clean.

Reply:
Thanks. I will work on it right away